### PR TITLE
Convert all top .html to .md

### PR DESCRIPTION
**Pull request by levitte**
Pandoc makes this really easy:

    pandoc -f html -t markdown -o index.html index.md

Only very minute small manual adjustments are needed afterwards.

Partial fix of #324

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Welcome to OpenSSL!
+
+The OpenSSL Project develops and maintains the OpenSSL software - a robust,
+commercial-grade, full-featured toolkit for general-purpose cryptography and
+secure communication. The project\'s technical decision making is managed by
```
Comment (paulidale) [inline index.md]: Ditch the \\

Comment (levitte) [reply index.md]: Done

**Review by paulidale — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (levitte) [reply index.md]: Done

**Comment by levitte** [comment]
I discovered that the inclusion of legalities.shtml was inadvertently dropped.  Fixed

**Comment by levitte** [comment]
All major glitches gone.  Final review, @openssl/omc?

**Comment by levitte** [comment]
Ping

**Review by paulidale — APPROVED** [review]

**Comment by levitte** [comment]
Merged

364ddb0bb1b9a4e922f9739a95ab149f04c893ec Convert all top .html to .md

**Pull request closed without merging**